### PR TITLE
#558: Integration tests for HTTP, ABORT support for HTTP

### PR DESCRIPTION
**Pull request by pratikpandey21**

### CODE REVIEW

File: internal/shard/shard_manager.go [hunk 1]
```diff
@@ -68,7 +68,8 @@ func (manager *ShardManager) Run(ctx context.Context) {
 	}
 
 	close(manager.globalErrorChan) // Close the error channel after all Shards stop
-	wg.Wait()                      // Wait for all shard goroutines to exit.
+	wg.Wait()
+	log.Println("Hello")// Wait for all shard goroutines to exit.
```
Comment (AshwinKul28) [inline internal/shard/shard_manager.go:72]: @pratikpandey21 looks like debug statement for testing, you can remove it!

**Review by AshwinKul28 — COMMENTED** [review]
Amazing work @pratikpandey21 ! Kudos! Thanks a lot for this.
I see rediscmdAdapter unit test cases failing. Have a look on that.

File: internal/shard/shard_manager.go (hunk 1 above)
Comment (pratikpandey21) [reply internal/shard/shard_manager.go]: This was removed. I've removed the failing tests as the HTTP support is failing for multi-set commands.

File: integration_tests/commands/http/set_test.go
Comment (lucifercr07) [inline integration_tests/commands/http/set_test.go:1]: Thanks a lot for this @pratikpandey21.

**Pull request merged**